yt/yt/library/auth_server/config.h

```diff
@@ -156,9 +156,25 @@ DEFINE_REFCOUNTED_TYPE(TCachingCypressTokenAuthenticatorConfig)
 
 ////////////////////////////////////////////////////////////////////////////////
 
-class TOAuthTokenAuthenticatorConfig
+class TOAuthAuthenticatorConfig
     : public virtual NYTree::TYsonStruct
 {
+public:
+    //! Creates a new user if it doesn't exist.
```

Comment:
Let's specify that it will create the user with the name from the `Login` field.